### ISSUE TITLE
ROS2 OCS does not launch behavior

### ISSUE THREAD

**Issue by drosen20** [FlexBE/flexbe_app#80]
I am experimenting with flexBE in ROS2 Galactic and I ran into an issue with launching a behavior from the OCS GUI to run on a remote machine.

What I did:
- followed the [Basic Tutorials](http://wiki.ros.org/flexbe/Tutorials) to create a hello world behavior that waits a few seconds and then prints to the screen
- launched the behavior from the GUI after running ```ros2 launch flexbe_app flexbe_full.launch.py``` and observed the expected behavior
- Copied the tutorials_behaviors repo (created in the tutorials) to the computer on my robot and ran ``colcon build`` to set up the workspace
- On remote, launched the OCS with ``ros2 launch flexbe_app flexbe_ocs.launch.py``. On robot, launched the behavior engine with ``ros2 launch flexbe_onboard behavior_onboard.launch.py``.

Results:
The OCS was able to connect to the behavior engine. After seeing --- Behavior Engine ready! --- on the robot the OCS indicated --- Behavior Mirror ready! ---. But after clicking Start Execution I get an error
```
Failed to retrieve behavior from library: [-]
787353483
Dropped behavior start request because preparation failed.
```

Also if I switch the launch order (engine on robot first, then OCS on remote), when I click Start Execution the OCS hangs on the Waiting For Behavior screen with the message Start command has been sent. Now waiting for confirmation from onboard behavior...

**Comment by drosen20** [comment]
With some poking around I discovered that the OCS had saved my hello world behavior somewhere in workspace/install instead of workspace/src/tutorials_behaviors/tutorials_flexbe_behaviors. Once I saved it there and added an xml file for the behavior to src/tutorials_behaviors/tutorials_flexbe_behaviors/manifest, I was able to launch the OCS first on remote, then the engine on the robot, and successfully execute the behavior.

Could I suggest modifying the default save location so that everything lives on the workspace/src side?
